src/BlazorWasmSdk/Tool/Program.cs

```diff
@@ -24,28 +24,28 @@ public static int Main(string[] args)
                 description: "System.IO.Compression.CompressionLevel for the Brotli compression algorithm.");
             var sourcesOption = new Option<List<string>>(
                 "-s",
-                description: "A list of files to compress.");
+                description: "A list of files to compress.") { AllowMultipleArgumentsPerToken = false };
             var outputsOption = new Option<List<string>>(
                 "-o",
-                "The filenames to output the compressed file to.");
+                "The filenames to output the compressed file to.") { AllowMultipleArgumentsPerToken = false };
 
             brotli.Add(compressionLevelOption);
             brotli.Add(sourcesOption);
             brotli.Add(outputsOption);
 
             rootCommand.Add(brotli);
 
-            brotli.Handler = CommandHandler.Create<CompressionLevel, List<string>, List<string>>((compressionLevel, sources, outputs) =>
+            brotli.Handler = CommandHandler.Create<CompressionLevel, List<string>, List<string>>((c, s, o) =>
```

Comment:
Why do we need this change? The previous variables looked a lot better.

Reply:
The `System.CommandLine` API matches variable names passed on the string you pass for the option in the `new Option` call. For example, if you create an option `new Option<string>('--intOptions')` then the variable name in the handler would be `intOptions`. An alternative solution to this would be to update the tool to take `--sources` and `--outputs` as options, update the Brotli compression task, and then use `sources` and `outputs` as variable names here. I figured this was the smallest delta.